streams/src/test/resources/kafka/kafka-streams-version.properties

```diff
@@ -0,0 +1,2 @@
+commitId=t35tc0441t
```

Comment:
This is just a fake commit id, right? It might be nice to make it self-descriptive, like "fakeCommitIdForTesting". Otherwise, it might get confusing when it shows up in the test logs.

Reply:
Done